dashboard/app/views/levels/editors/_gamelab.html.haml

```diff
@@ -1,3 +1,10 @@
+:scss
+  .field.teacher-only-markdown {
+    border: 5px solid gray;
+    border-radius: 5px;
+    padding: 10px;
+  }
```

Comment:
This is duplicated a few times, and feels like it should live somewhere else.  We have _some_ styling on these level edit pages.  Where does it live?